src/v0/destinations/google_adwords_remarketing_lists/networkHandler.js

```diff
@@ -89,21 +89,18 @@ const gaAudienceProxyRequest = async (request) => {
   // step1: offlineUserDataJobs creation
 
   const firstResponse = await createJob(endpoint, customerId, listId, headers, method);
-  if (!firstResponse.success && !isHttpStatusSuccess(firstResponse.response.response.status)) {
+  if (!firstResponse.success && !isHttpStatusSuccess(firstResponse?.response?.response?.status)) {
     return firstResponse;
   }
 
   // step2: putting users into the job
   let jobId;
-  if (
-    firstResponse?.response?.data &&
-    firstResponse?.response?.data?.resourceName
-  )
+  if (firstResponse?.response?.data && firstResponse?.response?.data?.resourceName)
```

Comment:
firstResponse?.response?.data?.resourceName only this sufficient right?

Reply:
Yeah, fixed it.